src/services/events.service.ts

```diff
@@ -53,9 +53,12 @@ export class EventsService {
 		}
 	}
 
-	static async list(limit?: number): Promise<ServiceResponse<BaseEvent[]>> {
+	static async list(limit?: number, destinationId?: string): Promise<ServiceResponse<BaseEvent[]>> {
 		try {
-			const { events } = await eventsClient.list({ maxResults: limit });
+			const { events } = await eventsClient.list({
+				maxResults: limit,
+				destinationId,
```

Comment:
```suggestion
				sourceId,
```

Reply:
It's impossible 
![image](https://github.com/user-attachments/assets/1cd71c1e-a461-4012-bebb-4981368a4ff1)

Reply:
Ok